dockerfiles/piraeus-server/entry.sh

```diff
@@ -19,8 +42,28 @@ run_migration() {
 			kubectl get "${CRD}" -oyaml > "${CRD}.yaml"
 		done
 		tar -czvf backup.tar.gz -- *.yaml
-		kubectl create secret generic "${BACKUP_NAME}" --type piraeus.io/linstor-backup --from-file=backup.tar.gz
-		kubectl annotate secrets "${BACKUP_NAME}" "piraeus.io/linstor-version=${VERSION}"
+
+		if ! create_backup_secret "${BACKUP_NAME}" backup.tar.gz "${VERSION}"; then
+			echo "===============================================================================" >&2
+			echo "Backup backup.tar.gz too large, even after chunking, to fit into secrets." >&2
+			echo "Please manually copy it to a safe location, by running:" >&2
+			echo "" >&2
+			echo "  kubectl cp -c run-migration $(hostname):/run/migration/backup.tar.gz ." >&2
+			echo "" >&2
+			echo "This container will wait until downloading of the backup has been confirmed." >&2
+			echo "To confirm the backup has been stored locally, create a secret using:" >&2
+			echo "" >&2
+			echo "  kubectl create secret generic ${BACKUP_NAME} --type piraeus.io/linstor-backup" >&2
+			echo "" >&2
```

Comment:
a subshell with 1 redirect might be easier to read? like

```bash
(
  echo "foo"
  echo "bar"
) >&2
```

or some "cat EOF" variant?